Slider-of-pictures/test.js

```diff
@@ -0,0 +1,32 @@
+const wrapImg = document.querySelector(".wrap-img");
```

Comment:
```suggestion
const wrapImage = document.querySelector(".wrap-img");
```
letters in js are free =)